Hold-Krykke/Security_and_FullstackJavascript_Exam
# Logging user in, if token in SecureStore

- [x] Logging user in, if token in secureStore 
- [x] Remove token from user, since its now in SecureStore 
- [x] A lot of debugging

**How to test**
1. Run backend, frontend, ngrok, replace ngrok yada yada
2. Log in
3. Close app
4. Reopen app
5. **Expected Behavior:** Should still be logged in

Right now we control the logged in state via a useState. And that is fine. But, since user might have a JWT in the SecureStore, we could make a useEffect, that checks for a token in securestore, and if it is there, then decodes it, and logs the user in? So they keep their logged in, even if they close Expo and the App? :) Could be Cool imo

https://github.com/Hold-Krykke/Security_and_FullstackJavascript_Exam/issues/23
